ci: added support for Ubuntu 22.04 LTS

ShubhamTatvamasi opened the pull request:
Signed-off-by: Shubham Tatvamasi <shubhamtatvamasi@gmail.com>

andrepuschmann commented:
Thanks for that @ShubhamTatvamasi. Do you mind changing the commit message to match the naming scheme we use for the repo, i.e. use some similar to `ci: added support for Ubuntu 22.04 LTS`?

Thanks

ShubhamTatvamasi commented:
> Thanks for that @ShubhamTatvamasi. Do you mind changing the commit message to match the naming scheme we use for the repo, i.e. use some similar to `ci: added support for Ubuntu 22.04 LTS`?
> 
> Thanks

Just updated the commit message.

andrepuschmann commented:
Pulled this commit in for the next release. Thanks again. Closing PR
